### DISCUSSION TITLE
libatk-1.0.so.0: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by playerdefault** [foxglove/community#967] [Bugs]
**Description**
Freshly installed Foxglove Studio is not launching. Seeing nothing. When I try to launch it from the command line via `foxglove-studio`, I get the following error: 
```
/snap/foxglove-studio/90/foxglove-studio: error while loading shared libraries: libatk-1.0.so.0: cannot open shared object file: No such file or directory
```

- Version: 1.87.0 (snap)
- Platform: Ubuntu 22.04 LTS
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native): N/A

**Comment by linear** [comment]
<p><a href="https://linear.app/foxglove/issue/FG-6909/libatk-10so0-cannot-open-shared-object-file-no-such-file-or-directory">FG-6909 libatk-1.0.so.0: cannot open shared object file: No such file or directory</a></p>

**Comment by jtbandes** [comment]
Could you try installing from the .deb and see if that behaves differently?

**Comment by playerdefault** [comment]
> Could you try installing from the .deb and see if that behaves differently?

issue is not occurring with the deb version

**Comment by defunctzombie** [comment]
We've been unable to reproduce this with v2.1 snap package. Seems like the deb is working for you so I will close this out. If you want to try the snap again and can provide steps to reproduce we can look into it again.
